index.js

```diff
@@ -8,9 +10,32 @@ let numberToShow = 0;
 // Read the history file and parse the data
 
 function readFile() {
-  const array = fs.readFileSync("testHistory.txt").toString().split("\n");
+  const homeDir = os.homedir();
+  const historyPath = path.join(homeDir, ".bash_history");
+  const possiblePath = [
```

Comment:
Can we use a set here? Not sure if that helps, might be good if we ever have duplicate paths. For now, this is fine.